Client/Frontend/Strings.swift

```diff
@@ -3772,20 +3772,20 @@ extension String {
             value: "Show products recommended by %@",
             comment: "Label of the switch from settings card displayed in the shopping review quality bottom sheet. The placeholder will be replaced with the app name.")
         public static let SettingsCardTurnOffButton = MZLocalizedString(
-            key: "Shopping.SettingsCard.TurnOff.Button.v120",
+            key: "Shopping.SettingsCard.TurnOffButton.Title.v120",
```

Comment:
This seems to be updating the string ID, whereas the actual string does not change?

Reply:
That was my understanding regarding [this comment](https://github.com/mozilla-l10n/firefoxios-l10n/pull/205#discussion_r1364772900). Changing the key here is not an issue since this is the first time we export the strings.
Was the comment intended for a different change?

Reply:
got it, that works